build.sbt

```diff
@@ -21,7 +21,8 @@ val collections =
       name := "collection-strawman",
       libraryDependencies ++= Seq(
         "org.scala-lang.modules" %% "scala-java8-compat" % "0.8.0",
-        "com.novocode" % "junit-interface" % "0.11" % Test
+        "com.novocode" % "junit-interface" % "0.11" % Test,
+        "org.scalatest" %% "scalatest" % "3.0.1" % Test
```

Comment:
I’m not sure we can afford such a dependency, especially if our goal is to eventually move to the scala/scala repository.